### PR TITLE
Replace var keyword with let/const

### PR DESCRIPTION
**Pull request by codeinvoke7**
I changed the var declaration to let/const where necessary

Fixes #1086 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updates
* [x] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by codeinvoke7** [comment]
Hello @TildaDares This the my contribution on issue assigned to me. I do kindly await your review and feedback. Thanks

File: src/edit/actions/OpacityAction.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 L.OpacityAction = L.EditAction.extend({
   initialize(map, overlay, options) {
-    var edit = overlay.editing;
-    var mode = edit._mode;
-    var use;
-    var tooltip;
+    let edit = overlay.editing;
```
Comment (PeculiarE) [inline src/edit/actions/OpacityAction.js:3]: The 'const' keyword should be used here rather than 'let'.

**Review by PeculiarE — CHANGES_REQUESTED** [review]

File: src/edit/actions/OpacityAction.js (hunk 1 above)
Comment (codeinvoke7) [reply src/edit/actions/OpacityAction.js]: @PeculiarE I have just applied the changes, pls do review again thanks

**Review by PeculiarE — APPROVED** [review]
Nice job! @codeinvoke7

@TildaDares....I believe this can be merged as well. Thank you!

**Comment by TildaDares** [comment]
Great work everyone!

**Comment by codeinvoke7** [comment]
> Great work everyone!

Thanks my mentor

**Pull request merged**